src/main/java/com/hyperwallet/clientsdk/util/HyperwalletEncryption.java

```diff
@@ -310,9 +358,31 @@ public HyperwalletEncryptionBuilder jwsExpirationMinutes(Integer jwsExpirationMi
             return this;
         }
 
+        public HyperwalletEncryptionBuilder proxy(Proxy proxy) {
+            this.proxy = proxy;
+            return this;
+        }
+
+        public HyperwalletEncryptionBuilder proxyUsername(String proxyUsername) {
+            this.proxyUsername = proxyUsername;
+            return this;
+        }
+
+        public HyperwalletEncryptionBuilder proxyPassword(String proxyPassword) {
+            this.proxyPassword = proxyPassword;
+            return this;
+        }
+
         public HyperwalletEncryption build() {
-            return new HyperwalletEncryption(encryptionAlgorithm, signAlgorithm, encryptionMethod,
+            HyperwalletEncryption hwE = new HyperwalletEncryption(encryptionAlgorithm, signAlgorithm, encryptionMethod,
                     clientPrivateKeySetLocation, hyperwalletKeySetLocation, jwsExpirationMinutes);
+            if (proxy != null)
```

Comment:
Minor: should be the following, make sure to run code formatting before commits (see https://engineering.paypalcorp.com/confluence/display/Payouts/IntelliJ+Setup)

```
if (proxy != null) {
                hwE.setProxy(proxy);
            }
```